Hawkins/template/README.md

```diff
@@ -3,11 +3,12 @@ functions that do stuf and thats:
 
     >>> from simple_functions import factorial
     >>> factorial(10)
-    9
-
-and this other part does something.  I forget why that I did it:
+    3628800
+	>>> factorial(5)
+    120
+and this other part does computes the factorial of the input. The second function determines the fibonnaccci sequence up to the input number
```

Comment:
Maybe you could clarify this by saying 'displays the Fibonacci sequence up to and including the largest number not larger than the input number

Reply:
Perfect, thanks for the suggestion. Adding it now

Reply:
> Also, currently if you enter a number larger than 360, the resulting hour will be >24. Do we want to ensure the output to be mod 24?

Add this as a statement to search for when angle input is >360 or <0